src/v2i-hub/DmsPlugin/src/dmsNTCIP.cpp

```diff
@@ -1466,7 +1473,8 @@ netsnmp_pdu*  SignalControllerNTCIP::setSNMPInt(const char* setOID, const char *
 			{
 				int *aa;
 				aa = (int *) vars->val.integer;
-				out[i++] = *aa;
+				out[i] = *aa;
+				i++;
 				//printf("value #%d is NOT a string! Ack!. Value = %d \n", count++,*aa);

```

Comment:
If the print statement is commented out we should either remove it or call plugin log macros right?